frontend/src/components/datasets/mainpage/SideCard.vue

```diff
@@ -6,31 +6,36 @@
 				<img v-else :src="dataset.thumbnail">
 			</div>
 			<h4>Details:</h4>
-			<p><strong>Year Created:</strong> {{dataset.year_created}}</p>
-			<p><strong>Size:</strong> {{dataset.size}}</p>
-			<p><strong>Number of Categories:</strong> {{dataset.num_cat}}</p>
-			<p><strong>COVE Profile Last Updated:</strong> {{dataset.date_created | moment}}</p>
+			<p v-if="dataset.year_created != null"><strong>Year Created:</strong> {{dataset.year_created}}</p>
+			<p v-if="dataset.size != null"><strong>Size:</strong> {{dataset.size}}</p>
+			<p v-if="dataset.num_cat != null"><strong>Number of Categories:</strong> {{dataset.num_cat}}</p>
+			<p><strong>COVE Profile Last Updated:</strong> {{dataset.last_updated | moment}}</p>
 						
-			<div class="tag-list">
+			<div class="tag-list" v-if="tasks.length > 0">
 				<p><strong>Tasks:</strong></p>
 				<div v-for="tag in tasks">
 					<router-link tag="a" :to="{name: 'home', query: {tasks: tag.name}}">{{tag.name}}</router-link>
 				</div>
 			</div>
 
-			<div class="tag-list">
+			<div class="tag-list"  v-if="topics.length > 0">
 				<p><strong>Topics:</strong></p>
 				<div v-for="tag in topics">
 					<router-link tag="a" :to="{name: 'home', query: {topics: tag.name}}">{{tag.name}}</router-link>
 				</div>
 			</div>
 
-			<div class="tag-list">
+			<div class="tag-list"  v-if="dataTypes.length > 0">
 				<p><strong>Data Types:</strong></p>
 				<div v-for="tag in dataTypes">
 					<router-link tag="a" :to="{name: 'home', query: {data_types: tag.name}}">{{tag.name}}</router-link>
 				</div>
 			</div>
+
+			<div id="ownership-request" v-if="dataset.is_owned_by_admin">
```

Comment:
Currently the admin who owns the dataset also sees the "Request Ownership" link, which doesn't make sense since the admin already owns it. Might not be worth the effort to fix though.

Reply:
Yea didn't feel the need to add the logic for that.